playground/alex_bot.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+from random import choice
+from dataclasses import dataclass
+from typing import List, Tuple
+from board import Stone
+
+
+@dataclass
+class Window:
+    pattern: List[int]
+    position: List[Tuple[int, int]]
+
+
+class AlexBot:
+
+    def __init__(self, k: int = 6, m: int = 19, n: int = 19):
+        self.k = k
+        self.m = m
+        self.n = n
+
+    def put_stone(self, log: List[Stone]) -> Stone:
+        turn = "b" if (len(log) + 1) % 4 in [0, 1] else "w"
```

Comment:
That's fair point, but I'm not sure is it okay if the bot refers referee code (It feels like cheating the system).

Please note that my first intention is to make bot totally independent with referee.

Reply:
Then we might separate `turn_check` as a common function and let both refree and bot use the function, since refree and bot are using the same logic to check turns